platform/centec/libsaithrift-dev.mk

```diff
@@ -4,4 +4,4 @@ LIBSAITHRIFT_DEV_CENTEC = libsaithrift-dev_0.9.4_amd64.deb
 $(LIBSAITHRIFT_DEV_CENTEC)_SRC_PATH = $(SRC_PATH)/SAI
 $(LIBSAITHRIFT_DEV_CENTEC)_DEPENDS += $(LIBTHRIFT) $(LIBTHRIFT_DEV) $(THRIFT_COMPILER) $(CENTEC_SAI)
 $(LIBSAITHRIFT_DEV_CENTEC)_RDEPENDS += $(LIBTHRIFT) $(CENTEC_SAI)
-SONIC_DPKG_DEBS += $(LIBSAITHRIFT_DEV_CENTEC)
+#SONIC_DPKG_DEBS += $(LIBSAITHRIFT_DEV_CENTEC)
```

Comment:
no need to comment this line.

Reply:
accept

Reply:
my compile is failed caused by this line. the log said that there are no src/SAI derectory for pushd.
 and referenced from BCM, Mellenox who can pass build on master, i comment it.